MAUI/Segmented-Control/migration.md

```diff
@@ -0,0 +1,603 @@
+---
+layout: post
+title: Migrate from Xamarin to .NET MAUI Segmented (SfSegmented) control | Syncfusion
```

Comment:
.NET MAUI Segmented control  (SfSegmented).

Reply:
changed to .NET MAUI Segmented control (SfSegmented).